presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -56,7 +58,8 @@ public Partition(
             @JsonProperty("partitionVersion") Optional<Long> partitionVersion,
             @JsonProperty("eligibleToIgnore") boolean eligibleToIgnore,
             @JsonProperty("sealedPartition") boolean sealedPartition,
-            @JsonProperty("createTime") int createTime)
+            @JsonProperty("createTime") int createTime,
+            @JsonProperty("lastAccessTime") DateTime lastAccessTime)
```

Comment:
@highker @rschlussel, should we just use int type to be compatible with createTime here? Especially @rschlussel mentioned that we will use Java.Time instead of Joda.Time later.

Reply:
sure; let's use int